txbits/app/models/UserModel.scala

```diff
@@ -57,7 +57,7 @@ class UserModel(val db: String = "default") {
   }
 
   def addFakeMoney(uid: Long, currency: String, amount: BigDecimal) = DB.withConnection(db) { implicit c =>
-    if (Play.current.configuration.getBoolean("fakeexchange").get) {
+    if (globals.fakeExchange) {
```

Comment:
Moving this to a variable in Global.scala makes the unit test `not be able to add fake balance on real exchange` fail. Any idea why?